docs/source/quick-start.mdx

```diff
@@ -0,0 +1,109 @@
+# Quick start
+
+The Hugging Face Hub is the go-to place for sharing machine learning models, spaces, datasets, and metrics. `huggingface_hub` library helps you interact with the Hub without leaving your development environment. You can create and manage repositories easily and download and upload files from the Hub. 
+
+## Install the Hub library
```

Comment:
For consistency with https://huggingface.co/docs/transformers/main/installation I think we could also show how to install with extras for TF and PyTorch as well as installing from source.

Reply:
I don't think that's in the scope of a quick start guide. Also, if we say how to do for TF and PyTorch, then we also give the impression that the hub is only, or mainly for those libraries.

Reply:
I think if we want to, we can create a separate page with the additional installation options. But it's probably best to keep the quick start guide as lightweight as possible without presenting other options that might distract the user from getting started.

Reply:
Yes, sounds good!